packages/truffle-db/src/loaders/artifacts/test/sources/Migrations.json

```diff
@@ -65,8 +65,14 @@
+  "bytecode": {
+    "bytes": "608060405234801561001057600080fd5b50336000806101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908373ffffffffffffffffffffffffffffffffffffffff1602179055506102ae806100606000396000f3fe608060405234801561001057600080fd5b506004361061004c5760003560e01c80630900f01014610051578063445df0ac146100955780638da5cb5b146100b3578063fdacd576146100fd575b600080fd5b6100936004803603602081101561006757600080fd5b81019080803573ffffffffffffffffffffffffffffffffffffffff16906020019092919050505061012b565b005b61009d6101f7565b6040518082815260200191505060405180910390f35b6100bb6101fd565b604051808273ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff16815260200191505060405180910390f35b6101296004803603602081101561011357600080fd5b8101908080359060200190929190505050610222565b005b6000809054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff1614156101f45760008190508073ffffffffffffffffffffffffffffffffffffffff1663fdacd5766001546040518263ffffffff1660e01b815260040180828152602001915050600060405180830381600087803b1580156101da57600080fd5b505af11580156101ee573d6000803e3d6000fd5b50505050505b50565b60015481565b6000809054906101000a900473ffffffffffffffffffffffffffffffffffffffff1681565b6000809054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff16141561027f57806001819055505b5056fea165627a7a72305820fd10a0b0c646daa7f5ee5eac53b76f2ba70ca857df1ac9142366169abaa138490029",
+    "linkReferences":[]
```

Comment:
The issue here was that these artifacts were compiled using the old compile originally. With the old compile, this object looks different in the artifacts. Please let me know if I have this wrong somehow!

Reply:
The artifacts format shouldn't have changed at all with the new compile interface! Let me know if that's not true... that'd be a bug!

Reply:
Sorry, its been a while since I did this part! So this is actually _not_ the fixture for the artifacts, which can be found at `/loaders/artifacts/test/compilationSources`; it's a stand-in for what the new compile should return in my mock of it (see the loaders test file). We talked about this way back when I was figuring this part out and you said that artifacts != compilation output and that is what I was trying to reflect here. 

I realize this is totally confusing and maybe there is a better way (or at the very least a renaming of the file). The new compilation result returns bytecode in this format, right? Apologies for the confusion here.